tests/test-distribution/test-distribution-common/src/test/java/org/eclipse/jetty/tests/distribution/DistributionTests.java

```diff
@@ -426,7 +429,8 @@ public void testLog4j2ModuleWithSimpleWebAppWithJSP(String env) throws Exception
             int port = Tester.freePort();
             try (JettyHomeTester.Run run2 = distribution.start("jetty.http.port=" + port))
             {
-                assertTrue(run2.awaitConsoleLogsFor("Started oejs.Server@", START_TIMEOUT, TimeUnit.SECONDS));
+                assertTrue(run2.awaitConsoleLogsFor("Started oejs.Server@", START_TIMEOUT, TimeUnit.SECONDS),
+                    () -> String.join("\n", run2.getLogs()));
```

Comment:
or as simple as 
```
logs(run2)
```

Reply:
we should probably have something more generic for this such 
```
 assertBuildStart(START_TIMEOUT, TimeUnit.SECONDS);
```

which do the assert on logs content.

means 2 methods:
```
assertBuildStart()
assertBuildStart(time, TimeUnit)
```
first one using default values